src/api/resources/images.py

```
@@ -0,0 +1,106 @@
+        return obj.project.name
+
+
+class AllImagesPermission(permissions.BasePermission):
+    def has_permission(self, request, view):
+        profile = getattr(request.user, "profile", None)
+        if profile is None:
+            return False
+
+        return request.method in permissions.SAFE_METHODS
+
+    def has_object_permission(self, request, view, obj):
```

Ruff (0.13.1)

Unused method argument: `view`

(ARG002)